docs/release-promotion/common/uncategorized.md

```diff
@@ -0,0 +1,14 @@
+# A word on selecting partials in Ship-it
+
+When selecting partials in [Ship-it](http://ship-it.mozilla.org/), you need to make sure those specific releases lie within both
+the `candidates` and `releases` directories. This is because [release-runner](hg.mozilla.org/build/tools/file/tip/buildfarm/release/release-runner3.py) is
+running some sanity checks on each of the provided partials. Part of the checks make sure that the certain release has been
+fully pushed to `~candidates` directory and then copied-over into the `~releases` directory. Absent this and the [Ship-it](http://ship-it.mozilla.org/) interface
+will throw an error.
+
+# A note on rerunning action tasks
+
+Ocassionally action tasks fail for various reasons (e.g. scope issues). Whenever the fix is done and you're ready to
+rerun the action task, make sure you cancel the previous run first. Otherwise, you'll end up with duplicates tasks within
```

Comment:
It would be clarify this to be clear that you need to cancel the taskGroupId that matches the taskId of the action task. "cancel the previous run" makes it sound like you're cancelling the action task itself, not the group it creates.

Also, what happens when some tasks from that group have already run? IIRC, this is why we did new builds instead of rerunning action tasks for 59.0.

Reply:
I think you're right, this is more confusing than helpful. I think I should better kill this, what do you think?

Reply:
I think it's probably worth having something that recommends against rerunning action task if some tasks that it created have already ran.